### DISCUSSION TITLE
上传文件失败，nginx有设置最大文件限制

### DISCUSSION THREAD

**Discussion by JunXiaoRuo** [AlistGo/alist#705] [Q&A] [answered]
## 上传100多mb的文件显示这个
![image](https://user-images.githubusercontent.com/47996900/157191196-70be9677-0761-4ece-b032-1f70542b3bc0.png)

## nginx设置了最大的文件20000m
![image](https://user-images.githubusercontent.com/47996900/157190947-8d6dd9c9-810e-4321-b058-3d4362b5f88b.png)

**Comment by xhofe** [comment, marked as answer]
超时了。 上传带宽小的话 后台要上传很久，nginx默认超时时间60s。
